ingest/ingest_pipeline.py

```diff
@@ -92,7 +92,6 @@ class IngestPipeline(object):
     JSON_CONVENTION = (
         '../schema/alexandria_convention/alexandria_convention_schema.json'
     )
-    EXTERNAL_JSON_CONVENTION = 'gs://broad-singlecellportal-public/schema/alexandria_convention/2.0.1/alexandria_convention_schema.json'
```

Comment:
When we [last discussed this](https://github.com/broadinstitute/scp-ingest-pipeline/pull/90#discussion_r417334379), we noted:

> it wouldn't make sense for me to remove EXTERNAL_JSON_CONVENTION until manage_study.py was using the API endpoint to obtain the metadata convention.

Is that done?  If not, will this change cause a bug?

I'm not confident this will cause a bug, so this looks fine to me, but it seems worth asking.

Reply:
In discussing further with Eno, I realized EXTERNAL_JSON_CONVENTION was never incorporated into manage_study.py so, as you point out, its removal won't cause a bug.

Thanks for checking!